examples/dask-nvtabular-criteo-benchmark.py

```diff
@@ -76,16 +111,11 @@ def main(args):
         os.environ["UCX_TLS"] = UCX_TLS
 
     # Cleanup output directory
-    BASE_DIR = args.out_path
+    BASE_DIR = args.out_path[:-1] if args.out_path[-1] == "/" else args.out_path
```

Comment:
(just a nitpick) I know the use of `BASE_DIR` is not your choice, and I this is not something I feel "strongly" about, but it seems that this should be `base_dir` throughout (since it is not reall a "constant" nor an environment variable).